hardware/esp8266com/esp8266/libraries/ESP8266WebServer/src/ESP8266WebServer.h

```diff
@@ -83,6 +83,13 @@ class ESP8266WebServer
   String argName(int i);          // get request argument name by number
   int args();                     // get arguments count
   bool hasArg(const char* name);  // check if argument exists
+  
+  void collectHeaders(const char* headerKeys[], const size_t headerKeysCount); // set the request headers to collect
+  String header(const char* name);   // get request header value by name
```

Comment:
For example, would `const String&` be a better return type? (Will be tricky to handle the "not found" case, but we could make a single empty string to use for all "not found" return values.)